virtool/account/oas.py

```diff
@@ -123,6 +126,13 @@ class Config:
 
 
 class CreateAPIKeyResponse(BaseModel):
+    created_at: datetime
+    groups: List[GroupMinimal]
+    id: str
+    key: str
+    name: str
+    permissions: Permissions
+
```

Comment:
Needs to be in core as `APIKey`. Only example should be here. This applies to the rest of your API key response models.